drivers/py/pes/harmonic.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, args=None):
     def check_arguments(self):
         """ Function that checks the arguments required to run the driver """
         try:
-            k = list(map(float, self.args.split()))
+            k = list(map(float, self.args.split(",")))
```

Comment:
also unrelated?

Reply:
welcome to the far West Max.
here no random bugfix is left behind